mono_repo/lib/src/commands/pub.dart

```diff
@@ -131,5 +131,10 @@ final String _sdkDir = (() {
   return aboveExecutable;
 })();
 
-final String pubPath =
+final String _pubPath =
     p.join(_sdkDir, 'bin', Platform.isWindows ? 'pub.bat' : 'pub');
+
+/// If FLUTTER_HOME exists, use "$FLUTTER_HOME/bin/flutter[.bat]",
+/// otherwise use "flutter[.bat]".
+final String _flutterPath = p.join(Platform.environment['FLUTTER_HOME'] ?? '',
```

Comment:
This is not quite right.

If the key exists, you want to use the full path you defined – I think.

If it does not exist, you want the existing behavior. Right?

Reply:
You are right, the p.join was indeed adding the "bin" when it shouldn't.
I've done some more testing and found out that the full path to FLUTTER_HOME isn't required, it was actually sufficient to just add ".bat" to the "flutter" command in case it's already on the windows PATH.
So the proposed change is a lot smaller now.